pkg/pipelines/statustracker/deployment.go

```diff
@@ -17,8 +16,9 @@ import (
 )
 
 const (
-	operatorName   = "commit-status-tracker"
-	containerImage = "quay.io/redhat-developer/commit-status-tracker:v0.0.1"
+	operatorName         = "commit-status-tracker"
+	containerImage       = "quay.io/redhat-developer/commit-status-tracker:v0.0.1"
+	commitStatusAppLabel = "commit-status-tracker-operator"
```

Comment:
I thought we would use "commit-status-tracker" as the app name label?

Reply:
commit-status-tracker is the operator name